.github/ISSUE_TEMPLATE/enhancement_proposal.yaml

```diff
@@ -0,0 +1,17 @@
+name: Enhancement Proposal
+description: File an enhancement proposal
+labels: ["Type: Enhancement", "Status: Triage"]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Thanks for taking the time to fill out this enhancement proposal! Before submitting your issue, please make
+        sure there isn't already a prior issue concerning this. If there is, please join that discussion instead.
```

Comment:
I am not a fan of contractions in documentation but it is just my opinion. There is a `it's` in the first file as well. Is there a ruling on this in the documentation guidelines?

Reply:
Our [style guide](https://docs.ubuntu.com/styleguide/en/#contractions) has a list of contractions that we can use to achieve a conversational tone. Both are on here, so seems in line with our guidelines.